roles/helm_common/library/helm_cli.py

```diff
@@ -77,6 +78,9 @@ def install(self):
         namespace = self.params.get('namespace')
         if namespace:
             cmd.extend(['--namespace', namespace])
+        timeout = self.params.get('timeout')
```

Comment:
Could we add also --timeout to command to supports it (helm delete) ? Could be done as separated PR if needed

Reply:
It makes sense, I'll add it.